trackpy/plots.py

```diff
@@ -392,10 +392,10 @@ def plot_traj3d(*args, **kwargs):
 ptraj3d = plot_traj3d
 
 @make_axes
-def annotate(centroids, image, circle_size=None, color=None,
+def annotate(centroids, image, circle_size=None, label=False, color=None,
```

Comment:
please put new kwargs at the end of the list (think of the case where someone is using `annotate(cent, img, 5, 'b')` this is now silently (as in the code _runs_) but does something different).